src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,49 @@
 package core.basesyntax;
 
-public class WorkWithFile {
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
 
+public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        File newFile = new File(toFileName);
+        File initialFile = new File(fromFileName);
+        int supply = 0;
+        int buy = 0;
+        int result;
+
+        try (BufferedWriter bufferWriter = new BufferedWriter(new FileWriter(newFile));
+             BufferedReader bufferedReader = new BufferedReader(new FileReader(initialFile))) {
+            String bufferValue = bufferedReader.readLine();
+            StringBuilder writerStringBuilder = new StringBuilder();
+            String name = "";
+            String count = "";
+
+            while (bufferValue != null) {
+                name = bufferValue.split(",")[0];
+                count = bufferValue.split(",")[1];
+
+                if (name.equals("supply")) {
+                    supply += Integer.parseInt(count);
+                } else if (name.equals("buy")) {
```

Comment:
```suggestion
                } else {
```
we have only 2 possible operation for now so we can simplify this condition